SLHCUpgradeSimulations/Configuration/python/combinedCustoms.py

```diff
@@ -1,26 +1,63 @@
 import FWCore.ParameterSet.Config as cms
-from SLHCUpgradeSimulations.Configuration.postLS1Customs import customisePostLS1
+
+from SLHCUpgradeSimulations.Configuration.postLS1Customs   import customisePostLS1
 from SLHCUpgradeSimulations.Configuration.customise_mixing import customise_NoCrossing
-from SLHCUpgradeSimulations.Configuration.phase1TkCustoms import customise as customisePhase1Tk
-from SLHCUpgradeSimulations.Configuration.HCalCustoms import customise_HcalPhase1, customise_HcalPhase0
+from SLHCUpgradeSimulations.Configuration.phase1TkCustoms  import customise as customisePhase1Tk
+from SLHCUpgradeSimulations.Configuration.HCalCustoms      import customise_HcalPhase1, customise_HcalPhase0
 
-import SLHCUpgradeSimulations.Configuration.aging as aging
+from SLHCUpgradeSimulations.Configuration.gemCustoms import customise2019 as customise_gem2019
+from SLHCUpgradeSimulations.Configuration.gemCustoms import customise2023 as customise_gem2023
+from SLHCUpgradeSimulations.Configuration.me0Customs import customise as customise_me0
+from SLHCUpgradeSimulations.Configuration.rpcCustoms import customise as customise_rpc
+from SLHCUpgradeSimulations.Configuration.fixMissingUpgradeGTPayloads import fixRPCConditions
 
+import SLHCUpgradeSimulations.Configuration.aging as aging
 
 def cust_2017(process):
     process=customisePostLS1(process)
     process=customisePhase1Tk(process)
     #process=customise_HcalPhase0(process)
-    #process=fixRPCConditions(process)
     return process
 
 def cust_2019(process):
     process=customisePostLS1(process)
     process=customisePhase1Tk(process)
     process=customise_HcalPhase1(process)
-    #process=fixRPCConditions(process)
     return process
 
+def cust_2019WithGem(process):
+    process=cust_2019(process)
+    process=customise_gem2019(process)
+    return process
+
+def cust_2023(process):
```

Comment:
does it make sense to add this now with so many things commented out? (cust_2023 and cust_2023Muon).
It could be better just to add cust_2023MuonOnly so that there are no conflicts with others merging the code from SLHC.